CruzHacks/cruzhacks-2021-website
# Feature/error

Problem
=======
As a prospective hacker,
I would like useful error webpage
To understand when I've incorrectly navigated a website.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199387105632109/f)

Solution
========
To implement the Error section of the webpage, flexbox-based and relative/absolute positioning and component styling were used to replicate the provided Figma design with responsiveness. Specifically, a secondary image layer was integrate beneath the preliminary content layer to showcase appropriate lens flares and to position them correctly. In the content layer, error title, image, and button were laid out using margins, but may be revised to incorporate more flexbox-based positioning types, such as `space-evenly` or `space-between`.

Change summary:
---------------
* boiler-plated 404 page c4e516e
* added images and improved responsiveness 0c13baf
* fixed image layering 44f8418

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/error`
2. Start local development server using `yarn start`
3. Navigate to any URL not corresponding to the homepage (`/`) to verify that error section is designed and responds correctly.
4. Click the "let's go home" button to verify that all error pages are correctly redirected to the homepage.
